src/Arcus.Templates.WebApi/Program.cs

```diff
@@ -6,16 +6,49 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+#if Serilog
+using Serilog;
+using Serilog.Events;
+#endif
 
 namespace Arcus.Templates.WebApi
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static int Main(string[] args)
         {
+#if Serilog
```

Comment:
This part was inspired and recommended by the docs of Serilog. Not sure if this is fully what we wan.t

Reply:
I would throw in Application Insights, personally, as well

Reply:
Hmm, but then it would not be minimal.
But yeah, I'll add it anyway 😈

Reply:
The template aims at being battle-ready which is a must imo so let's go there indeed!

Reply:
Done.